safe/gui/tools/multi_exposure_dialog.py

```diff
@@ -588,12 +590,18 @@ def accept(self):
                         0, analysis.name)
                     detailed_group.setVisible(Qt.Checked)
                     add_impact_layers_to_canvas(analysis, group=detailed_group)
+
+                if self.iface:
+                    self.iface.setActiveLayer(
+                        self._multi_exposure_if.analysis_impacted)
```

Comment:
I think we should put this outside the `if generate_report` condition, just like the behaviour in single IF

Reply:
I guess only me is running this option. The checkbox is displayed only with developer mode is ON and you restarted QGIS before so I think it's quite specific

Reply:
I just checked the code, this line is already outside the `if generate_report` ...